src/netbase.h

```diff
@@ -39,6 +39,8 @@ class CNetAddr
 {
     protected:
         unsigned char ip[16]; // in network byte order
+        unsigned char ip_tor[41]; //for compatibility with onion v3 addresses
+        bool isv3; // v2 or v3 onion address
```

Comment:
bool variables in general start with 'f' in bitcoin code base. v3 is not clear here. Suggest to change to **fTorV3**